cmd/karavictl/cmd/role_delete.go

```diff
@@ -47,7 +47,10 @@ func NewRoleDeleteCmd() *cobra.Command {
 			matched := make(map[roles.Instance]struct{})
 			for _, v := range roleFlags {
 				t := strings.Split(v, "=")
-				r := roles.NewInstance(t[0], t[1:]...)
+				r, err := roles.NewInstance(t[0], t[1:]...)
+				if err != nil {
+					continue
```

Comment:
Should this fail instead of continuing?

Reply:
No, its deleting a role, if multiple roles provided, instead of failing for one wrong role, we shall delete the valid roles, that why continue

Reply:
Should we log a message stating that a specific role couldn't be deleted? If I provide 3 roles to delete and don't see any error message, I would assume all 3 were deleted even though that may not be the case.

Reply:
This karavictl command, I do not see example of we are logging partial error. even instance creation is success, but one of the role is not found out of 3, we aren't logging anything. We are reporting error if none of 3 role are not found!

Reply:
fixed: will return error and will not continue.